### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards> InputsWithValidation.json> Input.Toggle]: Voice access user is not able to access 'Accept' and 'Dismiss' control under Start time button.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7522]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Problem Description**
Test Environment:
OS Version: 22H2 (Build 25126.1000)
Application: Adaptive Cards UWP
Application Version: 3.0.2206.2001
Tool: Voice access

**Repro Steps:**

1. Turn on voice access.
2. Open Adaptive Cards UWP application.
3. Command: Voice access wake up'
4. Command: Show Numbers
5. Select particular number for 'InputsWithValidation.json'
6. Command: Move to 16
7. Command: Click
8. Try to access the 'Accept and dismiss' control through any name.
9. Observe the issue.

**Actual Result:**
Voice access user is not able to access 'Accept' and 'Dismiss' control under Start time button as only icons of 'Accept' and 'Dismiss' control is present instead of visual name.

**Expected Result:**
Voice access user should be able to access 'Accept' and 'Dismiss' control under Start time button. As with icon visual name of  'Accept' and 'Dismiss' control should also be present.

**User Impact:**
People with limited mobility (cerebral palsy, quadriplegia) who use Voice Access will get impacted as user will not be able to give any command related to above controls and accessing through numbers takes more time.

**MAS reference link:**
[[No Disruption of Accessibility Features - Liquid (microsoft.com)]](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.02.01#Zguide)

**Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/104483005/173440001-30638a0a-0e0f-44ff-b508-0edf8243bbaa.mp4

<img width="960" alt="MAS4 3 1 Visual name is not appearing for Accept under start time" src="https://user-images.githubusercontent.com/104483005/173440123-db4eab98-4157-41f9-8161-2a48b9767a0f.png">
<img width="960" alt="MAS4 3 1 Dismiss control name is not visually appearing" src="https://user-images.githubusercontent.com/104483005/173441356-fa620c77-0f8f-4d2a-b654-8bf4c4047ad6.png">

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vaishali1397,

My team uses the [TimePicker control from WinUI](https://learn.microsoft.com/en-us/windows/apps/design/controls/time-picker) to create the element you are referencing. Because of this, we are unable to add a visual label to the "Accept" and "Dismiss" buttons.

Please let me know if you have any questions, thanks!

**Comment by vaishali1397** [comment]
Hi @anna-dingler could you please let us know the POC for WinUI control, so that we can mark our bug as external to WinUI.

**Comment by vaishali1397** [comment]
Closing this bug as per below comment.


![image](https://user-images.githubusercontent.com/104483005/215486855-f6f40444-413c-46a3-b5fc-67b412bf3e0c.png)

**Comment by anna-dingler** [comment]
Hi @vaishali1397,

For WinUI 2 related bugs, you can reach out to @bkudiess

Thanks!
